tests/integration_tests/scheduler/conftest.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+import asyncio
+import os
+import sys
+from pathlib import Path
+
+import pytest
+
+from ert.scheduler.driver import Driver
+from ert.scheduler.event import FinishedEvent, StartedEvent
+
+
+@pytest.fixture
+def mock_openpbs(pytestconfig, monkeypatch, tmp_path):
+    if pytestconfig.getoption("openpbs"):
+        # User provided --openpbs, which means we should use the actual OpenPBS
+        # cluster without mocking anything.
+        return
+    mock_bin(monkeypatch, tmp_path)
+
+
+@pytest.fixture
+def mock_lsf(pytestconfig, monkeypatch, tmp_path):
+    if pytestconfig.getoption("lsf"):
+        # User provided --lsf, which means we should use the actual LSF
+        # cluster without mocking anything.""
+        return
+    mock_bin(monkeypatch, tmp_path)
+
+
+async def poll(driver: Driver, expected: set[int], *, started=None, finished=None):
```

Comment:
There is a version of this one in unit_tests/test_lsf_driver.py as well. Maybe that one can also use this common function.

Reply:
Are you sure? I don't see anything there.

Reply:
It is well hidden far down... 
https://github.com/equinor/ert/blob/f26a69942191141f1d29998f2fb4010655096502/tests/unit_tests/scheduler/test_lsf_driver.py#L268

Reply:
Oh, my branch was outdated. Will move `poll` to `tests.utils`.

Reply:
Maybe also add a docstring explaining its intended usage.